### PR TITLE
feat: provide addintional user information

### PR DESCRIPTION
**Pull request by adeojoemmanuel**

### CODE REVIEW

File: controllers/erpControllers/authJwtController.js [hunk 1]
```diff
@@ -72,10 +72,14 @@ exports.login = async (req, res) => {
         success: true,
         result: {
           token,
-          admin: {
+          admin: { 
             id: result._id,
             name: result.name,
+            role: result.email,
```
Comment (polymahh) [inline controllers/erpControllers/authJwtController.js:78]: can you extract the role?

**Review by polymahh — CHANGES_REQUESTED** [review]

File: controllers/erpControllers/authJwtController.js (hunk 1 above)
Comment (salahlalami) [reply controllers/erpControllers/authJwtController.js:78]: remove role

**Review by salahlalami — CHANGES_REQUESTED** [review]
remove role

**Comment by polymahh** [comment]
@salahlalami we do need role for the profile page
image

**Comment by salahlalami** [comment]
@polymahh ok, @adeojoemmanuel just keep role

**Pull request closed without merging**